lightsoff_api/__init__.py

```diff
@@ -67,6 +68,7 @@ class PlaceBody(BaseModel):
     phone_number: Optional[str]
     latitude: float
     longitude: float
+    contacted: int
```

Comment:
I don't think you need this attribute when creating the place ;)

Reply:
That's done !